lib/utils/session_vars.js

```diff
@@ -174,66 +178,156 @@ session_vars.delete_project_name = function () {
 const artifacts_path_key = `artifacts_path`;
 session_vars.save_artifacts_path_name = function ( path_name ) {
   /* Save a key with the name of the current artifacts folder in case it doesn't exist. */
-
-  // Ensure artifacts path file exists
-  if ( !fs.existsSync( runtime_config_path ) ) {
-    // If the file doesn't exist, make one because next we need to read it
-    fs.writeFileSync( runtime_config_path, JSON.stringify( {}, null, 2 ) );
-  }
-
-  // Get the json, changing it, and re-write the file from scratch
-  let json = JSON.parse( fs.readFileSync( runtime_config_path ) );
-  json[ artifacts_path_key ] = path_name;
-  let file = fs.writeFileSync( runtime_config_path, JSON.stringify( json, null, 2 ) );
-
-  if ( session_vars.get_debug() ) {
-    console.log( `ALKiln debug: Stored name of artifacts directory "${ path_name }" locally.` );
-  }
-  return file;
+  return session_vars.save_runtime_config_var({ key: artifacts_path_key, value: path_name });
+
+  // // Ensure artifacts path file exists
+  // if ( !fs.existsSync( runtime_config_path ) ) {
+  //   // If the file doesn't exist, make one because next we need to read it
+  //   fs.writeFileSync( runtime_config_path, JSON.stringify( {}, null, 2 ) );
+  // }
```

Comment:
Regarding this commented-out code and the rest of it too - just want a sanity check on what I've done and then I'll clean it up. There might be another one I can abstract too, I'm not sure.

Also, should all the spots using these specific methods just use `session_vars.save_runtime_config_var()` directly?

Reply:
I'll clean up and leave a comment to discuss later or something. Or maybe make a 'question' issue.

Reply:
Decision: We really shouldn't get rid of the other functions. They keep track of the right runtime config key and we don't want each location to need to know that.